autogen/variables.tf

```diff
@@ -422,4 +422,15 @@ variable "authenticator_security_group" {
   default     = null
 }
 
+variable "enable_release_channel" {
```

Comment:
Do we need a separate `ENABLE` variable here? I'm wondering if the default for `release_channel` could be to simply not enable it at all?

Reply:
Yes, you are right. Since release channel is defaulted to `UNSPECIFIED`, then it is assumed that release channels are not used.

Removed `enable_release_channel`

Reply:
I think we should have a `null` default and omit the entire block in that case.

Reply:
Done